auto_nag/scripts/not_landed.py

```diff
@@ -208,69 +168,70 @@ def handle_attachment(self, attachment, res):
     def get_patch_data(self, bugs):
         """Get patch information in bugs
         """
-        nightly_pats = Bugzilla.get_landing_patterns(channels=["nightly"])
-
-        def comment_handler(bug, bugid, data):
-            r = Bugzilla.get_landing_comments(bug["comments"], [], nightly_pats)
-            landed = bool(r)
-            if not landed:
-                for comment in bug["comments"]:
-                    comment = comment["text"].lower()
-                    if "backed out" in comment or "backout" in comment:
```

Comment:
You don't want to check this anymore?

Reply:
This code was just used for splinter patches and I removed splinter things.
I must check that it's ok with phab patch too